app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -99,103 +99,108 @@ import com.dessalines.thumbkey.keyboards.KB_UK_RU_MESSAGEEASE_SYMBOLS
+    RUMessageEaseSymbols(KB_RU_MESSAGEEASE_SYMBOLS),
+    T9(KB_T9),
+    JAThumbKeyHiragana(KB_JA_THUMBKEY_HIRAGANA),
+    JAThumbKeyKatakana(KB_JA_THUMBKEY_KATAKANA),
+    FRThumbKeyV2(KB_FR_THUMBKEY_V2),
+    SVThumbKey(KB_SV_THUMBKEY),
+    TRThumbKey(KB_TR_THUMBKEY),
+    ENTypeSplit(KB_EN_TYPESPLIT),
+    ESTypeSplit(KB_ES_TYPESPLIT),
+    DETypeSplit(KB_DE_TYPESPLIT),
+    FRTypeSplit(KB_FR_TYPESPLIT),
+    ITTypeSplit(KB_IT_TYPESPLIT),
+    PTTypeSplit(KB_PT_TYPESPLIT),
+    PLTypeSplit(KB_PL_TYPESPLIT),
+    ENTwoHands(KB_EN_TWO_HANDS),
+    ENThumbKeyProgrammerWide(KB_EN_THUMBKEY_PROGRAMMER_WIDE),
+    HUThumbKey(KB_HU_THUMBKEY),
+    ESEOThumbKey(KB_ES_EO_THUMBKEY),
+    ITMessageEase(KB_IT_MESSAGEEASE),
+    ENSKThumbKey(KB_EN_SK_THUMBKEY),
+    HEThumbKey(KB_HE_THUMBKEY),
+    EOENDEThumbKey(KB_EOENDE_THUMBKEY),
+    GRThumbKey(KB_GR_THUMBKEY),
+    ENCZThumbKey(KB_EN_CZ_THUMBKEY),
+    ESMessageEase(KB_ES_MESSAGEEASE),
+    RUMessageEase(KB_RU_MESSAGEEASE),
+    BGThumbKeySymbols(KB_BG_THUMBKEY_SYMBOLS),
+    HRTwoHands(KB_HR_TWO_HANDS),
+    HRThumbKey(KB_HR_THUMBKEY),
+    HRThumbKeySymbols(KB_HR_THUMBKEY_SYMBOLS),
+    FITypeSplit(KB_FI_TYPESPLIT),
+    LVLTGThumbKey(KB_LV_LTG_THUMBKEY),
+    LTThumbKey(KB_LT_THUMBKEY),
+    IDThumbKeySymbolsV2(KB_ID_THUMBKEY_SYMBOLS_V2),
+    IDThumbKeySymbolsNumeric(KB_ID_THUMBKEY_SYMBOLS_NUMERIC_V1),
+    ESCAThumbKey(KB_ES_CA_THUMBKEY),
+    ENITThumbKey(KB_EN_IT_THUMBKEY),
+    ENEOMessageEaseSymbols(KB_ENEO_MESSAGEEASE_SYMBOLS),
+    UKRUMessageEaseSymbols(KB_UK_RU_MESSAGEEASE_SYMBOLS),
+    DEMessageEaseSymbols(KB_DE_MESSAGEEASE_SYMBOLS),
+    CAThumbKey(KB_CA_THUMBKEY),
+    MATHThumbKey(KB_MATH_THUMBKEY),
+    ENEEThumbKey(KB_EN_EE_THUMBKEY),
+    SKThumbKey(KB_SK_THUMBKEY),
+    ENNOTypeSplit(KB_EN_NO_TYPESPLIT),
+    ENThumbKeyCompose(KB_EN_THUMBKEY_COMPOSE),
+    ENNOThumbKey(KB_EN_NO_THUMBKEY),
+    ENTwoHandsSymbols(KB_EN_TWO_HANDS_SYMBOLS),
+    ENTwoHandsSymbolsNumbers(KB_EN_TWO_HANDS_SYMBOLS_NUMBERS),
+    JATypeSplitHiragana(KB_JA_TYPESPLIT_HIRAGANA),
+    JATypeSplitKatakana(KB_JA_TYPESPLIT_KATAKANA),
+    EUESThumbKey(KB_EU_ES_THUMBKEY),
+    ENQwertyFour(KB_EN_QWERTYFOUR),
+    KZThumbKey(KB_KZ_THUMBKEY),
+    ENMIThumbKey(KB_EN_MI_THUMBKEY),
+    ENMIThumbKeySymbols(KB_EN_MI_THUMBKEY_SYMBOLS),
+    ESCAMessageEase(KB_ES_CA_MESSAGEEASE),
+    SVMessageEase(KB_SV_MESSAGEEASE),
+    ENDoubleSymbols(KB_EN_DOUBLE_SYMBOLS),
+    FIMessageEase(KB_FI_MESSAGEEASE),
+    ENDAThumbKeySymbols(KB_EN_DA_THUMBKEY_SYMBOLS),
+    FIEEMessageEase(KB_FI_EE_MESSAGEEASE_SYMBOLS),
+    ENPLThumbkey(KB_EN_PL_THUMBKEY),
+    BGPhoneticMessagEaseSymbols(KB_BG_PHONETIC_MESSAGEASE_SYMBOLS),
+    UKMessageEaseSymbols(KB_UK_MESSAGEEASE_SYMBOLS),
+    ENFRMessagEaseSymbols(KB_EN_FR_MESSAGEASE_SYMBOLS),
+    HEMessageEaseSymbols(KB_HE_MESSAGEEASE_SYMBOLS),
+    ENNOMessagEaseSymbols(KB_EN_NO_MESSAGEASE_SYMBOLS),
+
+    ;
+
+    val index: Int
+        get() = ordinal
```

Comment:
Good call! But you can just remove this pointless function, and do `.ordinal` where we used to use `index`

Reply:
done